src/localization/en.json

```diff
@@ -241,7 +241,11 @@
     "preview": "Questionnaire preview",
     "is": "is",
     "delete": "Delete Question",
-    "back": "Back"
+    "back": "Back",
+    "description": "Description",
+    "insertDescription": "Insert Description Here",
+    "countedAnswers": "True values in this Questionnaire:",
```

Comment:
I'm not sure what this string is actually indicating. Do you mean correct answers? Number of correct answers in this questionnnaire? That seems like an off string, surely the number of correct answers/ true values is the same as the number of questions?